ts/components/conversation/ConversationHeader.tsx

```diff
@@ -289,27 +303,61 @@ export class ConversationHeader extends React.Component<Props> {
     );
   }
 
+  public renderSelectionOverlay() {
+    const { onDeleteSelectedMessages, onCloseOverlay, i18n } = this.props;
+
+    return (
+      <div className="message-selection-overlay">
+        <div className="close-button">
+          <SessionIconButton
+            iconType={SessionIconType.Exit}
+            iconSize={SessionIconSize.Medium}
+            onClick={onCloseOverlay}
+          />
+        </div>
+
+        <div className="button-group">
+          <SessionButton
+            buttonType={SessionButtonType.Default}
+            buttonColor={SessionButtonColor.Primary}
+            text={i18n('forwardMessage')}
+          />
+
+          <SessionButton
+            buttonType={SessionButtonType.Default}
+            buttonColor={SessionButtonColor.Danger}
+            text={i18n('delete')}
+            onClick={onDeleteSelectedMessages}
+          />
+        </div>
+      </div>
+    );
+  }
+
   public render() {
     const { id, isGroup, isPublic } = this.props;
     const triggerId = `conversation-${id}`;
 
     const isPrivateGroup = isGroup && !isPublic;
 
     return (
-      <div className="module-conversation-header">
-        {this.renderOptions(triggerId)}
-        {this.renderBackButton()}
-        <div className="module-conversation-header__title-container">
-          <div className="module-conversation-header__title-flex">
-            {this.renderTitle()}
-            {isPrivateGroup ? this.renderMemberCount() : null}
+      <>
```

Comment:
uhm what's this for? If you want a parent component that doesn't render anything then you can just use `<React.Fragment>`